CLR/Graphics/Graphics.cpp

```diff
@@ -195,7 +195,7 @@ HRESULT CLR_GFX_Bitmap::CreateInstance( CLR_RT_HeapBlock& ref, const CLR_UINT8*
         /* When loading a Windows BMP, GIF, or JPEG file, it is converted in-place to the native BPP.
          * When loading a compressed TinyCLR Bitmap from a resource file, two bitmaps are needed to decompress, then convert.
          * This fragments the heap and wastes space until the next garbage collection is done.
-         * When using the SimpleHeap, there is no relocation, so decompressing into a temp bitmap into the simpleheap wastes
+         * When using the heap, there is no relocation, so decompressing into a temp bitmap into the heap wastes

```

Comment:
The large object heap was isolated for a reason, to allow the OEM to partition out memory used for LARGE objects. Throwing everything into one big heap may be a simple solution to port but can lead to fragmentation problems, which is why the large buffer and bitmap images were isolated into a separate heap. This might be something an OEM could manage using a custom implementation of malloc()/free() (e.g. use the size parameter of malloc to determine which underlying region to allocate from), but doing so is more difficult as replacing the standard functions is slightly less than intuitively obvious to the casual observer 😉 and compiler/tool specific. While I'm all for getting rid of custom heap use for the normal PAL/HAL work (that should stick with standard malloc for greater portability and re-use) I think it makes life easier for ports needing large buffers to still have this as an option.